### PR TITLE
Integration/auth

### PR DESCRIPTION
**Pull request by SantoshPhaiju**

### CODE REVIEW

File: package-lock.json
Comment (AbhishekSunar01) [inline package-lock.json:1]: Do not use npm
image
image
we dont have package-lock.json file on main branch

File: src/services/api/auth.ts [hunk 1]
```diff
@@ -1,11 +1,40 @@
+import { AxiosResponse } from "axios";
+
 import { axiosInstance } from ".";
-import { AdminLogin } from "../../types/type";
+import { ILoggedInUserResponse, ILoginResponse } from "../../types/authTypes";
+import Cookies from "js-cookie";
+
+export const handleLogin = async ({
+  email,
```
Comment (AbhishekSunar01) [inline src/services/api/auth.ts:8]: Its better to use types here

**Review by AbhishekSunar01 — COMMENTED** [review]
Delete the package-lock.json

File: src/services/api/auth.ts (hunk 1 above)
Comment (SantoshPhaiju) [reply src/services/api/auth.ts:8]: okok🙂

File: package-lock.json
Comment (SantoshPhaiju) [reply package-lock.json:1]: done

**Review by AbhishekSunar01 — APPROVED** [review]

**Pull request merged**